pkg/types/types.go

```diff
@@ -54,6 +54,8 @@ type IPAMConfig struct {
 	OmitRanges               []string             `json:"exclude,omitempty"`
 	DNS                      cnitypes.DNS         `json:"dns"`
 	Range                    string               `json:"range"`
+	NodeSliceFastRange       string               `json:"nodeSliceFastRange"`
```

Comment:
need inputs on these fields, thinking we just reuse the existing Range field, add the node_slice_fast_range field and then we initially don't support rangestart/rangeend/overlapping_ranges in this mode